contents/blog/posthog-alternatives.md

```diff
@@ -0,0 +1,727 @@
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center">Open core, build your own integrations</td>
+   		</tr>
+    	<tr>
+        	<td>Cute hedgehogs</td>
+      		<td className="text-center"><span className="text-red text-lg">✖</span></td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center">Joy</td>
+   		</tr>
+	</tbody>
+</table>
+</div>
+
+### Key differences
+
+1. **Value and transparency:** FullStory offers no free plan and is notoriously pricey. PostHog offers a generous free plan (1 million events and 15k recordings per month), and all pricing is transparent. 
+
+2. **Heat maps and scroll maps:** PostHog supports click maps via the [PostHog Toolbar](/docs/product-analytics/toolbar), but FullStory goes further in offering cursor heatmaps, and scroll tracking, making it more useful for UX researchers.
+
+3. **Use cases:** FullStory is a session replay tool first – it's ideal for UX researchers and customer support. PostHog is useful for UX researchers and customer support too, but it's primarily designed for technical users (e.g. engineering and product teams) rather than support.
+
+**Read more:** [In-depth PostHog vs FullStory comparison](/blog/posthog-vs-fullstory) 
+
+### What people like about FullStory
+
+Customer experience and UX researchers like FullStory for troubleshooting:
+
+- "I like how I can look specifically into a feature to figure out how users are actually using it. It's also great for debugging and seeing issues with flows."
+
+- "It's been indispensable for our customer service and our tech teams and has helped us understand user behavior."
+
+- "Easiest way to handle bug replication and discovery as the full session recordings are key for engineers."
+
+Autocapture is also popular:
+
+- "A huge benefit is not requiring development work to set up new events or track site behavior as fullstory does this automatically."
+
+### What people dislike about FullStory
+
+High costs and paywalled features are frequently complaints:
+
+- "Due to the high subscription prices, we're forced to pick and choose which sections we want to track and when to turn Full Story on or off."
+
+- "The plan we're on limits us to a specific number of sessions, and we sometimes run out before the end of the month. The cost of jumping up to the next plan is very significant so we end up running out of sessions."
+
+- "FullStory not only charges more than leading competitors, but it also charges a lot more for premium features like conversions, journeys and detections. Plus, beware of their contract renewal terms; it can be confusing."
+
+- "There are a lot of features that are premium that feel like they should be included in the normal subscription."
+
+- "Since they have restructured their payment tiers, doubling our sessions will require increasing our cost by over 2x."
```

Comment:
```suggestion
- "Due to the high subscription prices, we're forced to pick and choose which sections we want to track and when to turn Full Story on or off."

- "FullStory not only charges more than leading competitors, but it also charges a lot more for premium features like conversions, journeys and detections. Plus, beware of their contract renewal terms; it can be confusing."
```